### PR TITLE
feat: add default layout

### PR DESCRIPTION
**Pull request by JheysonSaavedr4**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Ci
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]+.[1-9]'
```
Comment (danielsolartech) [inline .github/workflows/ci.yml:10]: Pasa lo mismo que con la #28

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  pull_request:
+    branches:
+      - '*'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]+.[1-9]'
```
Comment (danielsolartech) [inline .github/workflows/ci.yml:18]: Igual acá (#28 )

File: package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "nextchat-web",
   "version": "1.0.0-alpha",
+  "license": "GPL-3.0-or-later",
```
Comment (danielsolartech) [inline package.json:4]: Lo mismo #28

**Review by danielsolartech — CHANGES_REQUESTED** [review]

**Comment by JheysonSaavedr4** [comment]
Invalid, difference of commits between dev branch and pr. Will be re-created.

**Pull request closed without merging**